boards/xen/xenvm/xenvm.yaml

```diff
@@ -7,3 +7,6 @@ toolchain:
   - cross-compile
 ram: 16384
 vendor: xen
+testing:
```

Comment:
Is d201301 meant to be in this PR? It is discussed in separate PR #70129
Probably spurious push